example_tracepoint_test.go

```diff
@@ -0,0 +1,110 @@
+// +build linux
+
+package ebpf_test
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"runtime"
+	"time"
+
+	"github.com/cilium/ebpf"
+	"github.com/cilium/ebpf/asm"
+	ringbuffer "github.com/cilium/ebpf/perf"
+
+	"github.com/elastic/go-perf"
```

Comment:
I guess this means the package will be included in the go.mod? Probably acceptable.

Reply:
yes. It adds `github.com/elastic/go-perf` with the `indirect` tag in `go.mod`. Added this as well now.